app/src/main/java/com/github/se/eventradar/model/message/MessageHistory.kt

```diff
@@ -29,7 +29,6 @@ data class MessageHistory(
     map["latest_message_id"] = latestMessageId
     map["from_user_read"] = user1ReadMostRecentMessage
     map["to_user_read"] = user2ReadMostRecentMessage
-    map["messages"] = messages.map { it.toMap() }
```

Comment:
This change causes a test in `FirebaseMessageRepositoryUnitTest.kt` to fail.
What is your reasoning?

Reply:
I don't think that messages should be updated when converting message history to map. It depends on how it is done on the firebase side, but if messages are automatically added to firebase when adding messages, then this is not needed.

Reply:
Actually this is now handled in **addMessage()**. I kept this change and fixed the failing test.